modules/integration/tests-integration/tests-patches/src/test/resources/artifacts/ESB/sequence/esbjava4565/testSequence.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~
+  ~ WSO2 Inc. licenses this file to you under the Apache License,
+  ~ Version 2.0 (the "License"); you may not use this file except
+  ~ in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied. See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+<sequence xmlns="http://ws.apache.org/ns/synapse" name="testSequence">
+    <log>
+        <property name="testSequenceLog" value="testSequence called..."/>
+    </log>
+    <property
+            expression="get-property('registry','gov:/services/test/config/ftp.xml')"
+            name="ftpUrl" scope="default" type="OM"/>
```

Comment:
This is the auto formatted xml by the idea with standard wso2 formatting guidelines. Attributes in property tag placed in below line as it is exceed the maximum column length.